src/modm/platform/gpio/stm32/module.lb

```diff
@@ -199,6 +216,19 @@ def validate(env):
                 raise ValidateException("Pin '{}' already in EXTI map!".format(str(num)))
             extimap[str(num)] = vec
 
+    # Compute the set of remapped pins
+    positions = defaultdict(list)
+    for p in driver["package"][0]["pin"]:
+        if p.get("type", "io") == "io":
+            name = p["name"][1:4].strip().lower()
+            if len(name) > 2 and not name[2].isdigit():
+                name = name[:2]
+            positions[p["position"]].append( (name, p.get("variant", "") == "remap") )
+    # Only choose the GPIOs that are on the same package position and have remap name format
+    remapped_gpios = {pin[0]: pin[1]
+                          for pins in positions.values() if len(pins) > 1
+                              for pin in pins  if ("[" in pin[0] and "]" in pin[0])}
```

Comment:
This is a bit inelegant, I'll check if I can instead improve the modm-devices format here.

Reply:
F042 has no `[...]` in the remapped pin names:
`<pin position="17" name="PA11" variant="remap"/>`

Reply:
Well, now I have to fix it in modm-devices 😛

Reply:
Fixed it by explicitly marking the remap pin pair. Also makes it much easier to process in modm and works with the F042 format.